fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -385,8 +387,9 @@ protected Action buildActionChain() {
             ParameterDAO parameterDao = makeParameterDAO(connection);
 
             // Persist the Resource DTO.
+            final String parameterHashB64 = "";
             resourceDao.setPersistenceContext(context);
-            resourceDao.insert(resourceDTO, this.extractSearchParameters(updatedResource, resourceDTO), parameterDao);
+            resourceDao.insert(resourceDTO, this.extractSearchParameters(updatedResource, resourceDTO), parameterHashB64, parameterDao);
```

Comment:
The idea here is that Troy's PR will come through and start to actually populate these, right?

Reply:
Correct.